examples/js/controls/OrbitControls.js

```diff
@@ -133,12 +133,7 @@ THREE.OrbitControls = function ( object, domElement ) {
 			offset.applyQuaternion( quat );
 
 			// angle from z-axis around y-axis
-
-			theta = Math.atan2( offset.x, offset.z );
-
-			// angle from y-axis
-
-			phi = Math.atan2( Math.sqrt( offset.x * offset.x + offset.z * offset.z ), offset.y );
+			spherical.fromVector3( offset );
```

Comment:
Given that I needed to calculate radius anyhow, I changed the form to an equivalent that requires less mathematical operations.  It gives the same answers, and it only fails if radius = 0, but the other form also failed at that edge case.  I explicitly protected against that edge case in the code.